### ISSUE TITLE
feat (YouTube): `Hide view all posts button`

### ISSUE THREAD

**Issue by ILoveOpenSourceApplications** [ReVanced/revanced-patches#2757]
### Feature description

Instead of the comments box under community posts, for some posts there's a view all posts button which when clicked upon will take the user to a page full of posts from random channels (almost like Instagram feed).

View all posts button:

![Screenshot_20240221_183746_YouTube~2](https://github.com/ReVanced/revanced-patches/assets/117499019/df8da837-b1e4-4b50-b93d-351550119b6c)

### Motivation

It's weird to have a page full of posts instead of the occasional posts from the pages you've subscribed to as you're being taken to a page full of posts from random channels.

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
Try finding the litho component name, I don't have this button on my end

**Comment by ILoveOpenSourceApplications** [comment]
I did try to capture the litho component but the log didn't show anything which hides the button only in specific.

**Comment by oSumAtrIX** [comment]
Please send the relevant log

**Comment by ILoveOpenSourceApplications** [comment]
I captured the log with protobuffer turned off.

**Comment by ILoveOpenSourceApplications** [comment]
```kotlin
02-21 18:00:47.620  2700  2981 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|post_engagement_toolbar.eml|ddc981ccdd1c3fd9|ContainerType|ContainerType|post_comment_button.eml|285fe3c99a0f9574|ContainerType|post_comment_count.eml|4d9bf50cf465aac1|ContainerType|TextType|
```

**Comment by oSumAtrIX** [comment]
This line is only the component before the button

**Comment by ILoveOpenSourceApplications** [comment]
```kotlin
02-21 23:46:44.696 16424 22626 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|ContainerType|
02-21 23:46:44.696 16424 22626 D revanced: LithoFilterPatch: BufferStrings: post_menu.eml|cf95372aa71af81f❙>%G2❙Key_PostMetadataWithController❙post_metadata.eml|8fdafefdca415443❙&K}6❙post_avatar.eml|32d70da7bae26a07❙

```

**Comment by oSumAtrIX** [comment]
As the string says, that's the components at the top

**Comment by ILoveOpenSourceApplications** [comment]
```kotlin
02-21 23:45:16.205 16424 22627 D revanced: Filter: AdsFilter Filtered path: full_width_square_image_layout.eml|e0282617afc591fa|CellType|
02-21 23:45:16.637 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.637 16424 16424 D revanced: LithoFilterPatch: ID: post_base_wrapper.eml|5e2b195a9a5d7880
02-21 23:45:16.637 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|
02-21 23:45:16.637 16424 16424 D revanced: LithoFilterPatch: BufferStrings: expand-post-cell❙post_base.eml|397e38aba64a8981❙&g_{❙@EiRVZ2t4ZjJvQVBjUjB4OHlqR1F1QmlMbmtjRUVZcExUOTRTUmIgqAIoAQ%3D%3D❙".B,❙1708539316247679070❙post_base_wrapper.eml|5e2b195a9a5d7880❙
02-21 23:45:16.637 16424 16424 D revanced: FilterGroupList: Creating prefix search tree for: app.revanced.integrations.youtube.patches.components.StringFilterGroupList@a8990f1
02-21 23:45:16.639 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.639 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.639 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|
02-21 23:45:16.639 16424 16424 D revanced: LithoFilterPatch: BufferStrings: expand-post-cell❙post_base.eml|397e38aba64a8981❙&g_{❙@EiRVZ2t4ZjJvQVBjUjB4OHlqR1F1QmlMbmtjRUVZcExUOTRTUmIgqAIoAQ%3D%3D❙".B,❙1708539316247679070❙post_base_wrapper.eml|5e2b195a9a5d7880❙
02-21 23:45:16.643 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.643 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.643 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|
02-21 23:45:16.643 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_Post❙post.eml|b9dd6295ab418a5f❙FpJ5❙id.ui.backstage.post❙
02-21 23:45:16.643 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.643 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.643 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|
02-21 23:45:16.643 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_Post❙post.eml|b9dd6295ab418a5f❙FpJ5❙id.ui.backstage.post❙
02-21 23:45:16.645 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.645 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.645 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|
02-21 23:45:16.645 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_Post❙post_engagement_toolbar_transition_key❙post_engagement_toolbar.eml|ddc981ccdd1c3fd9❙post_content.eml|e73333795cadcc57❙post_header.eml|2dce4b70ac02cd14❙id.ui.backstage.original_post❙
02-21 23:45:16.646 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.646 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.646 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|
02-21 23:45:16.646 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_Post❙post_engagement_toolbar_transition_key❙post_engagement_toolbar.eml|ddc981ccdd1c3fd9❙post_content.eml|e73333795cadcc57❙post_header.eml|2dce4b70ac02cd14❙id.ui.backstage.original_post❙
02-21 23:45:16.647 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.647 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.647 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|
02-21 23:45:16.647 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_Post❙post_engagement_toolbar_transition_key❙post_engagement_toolbar.eml|ddc981ccdd1c3fd9❙post_content.eml|e73333795cadcc57❙post_header.eml|2dce4b70ac02cd14❙id.ui.backstage.original_post❙
02-21 23:45:16.648 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.648 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.648 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|
02-21 23:45:16.648 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_Post❙post_engagement_toolbar_transition_key❙post_engagement_toolbar.eml|ddc981ccdd1c3fd9❙post_content.eml|e73333795cadcc57❙post_header.eml|2dce4b70ac02cd14❙id.ui.backstage.original_post❙
02-21 23:45:16.648 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.648 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.648 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|
02-21 23:45:16.648 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_Post❙post_engagement_toolbar_transition_key❙post_engagement_toolbar.eml|ddc981ccdd1c3fd9❙post_content.eml|e73333795cadcc57❙post_header.eml|2dce4b70ac02cd14❙id.ui.backstage.original_post❙
02-21 23:45:16.649 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.649 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.649 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|
02-21 23:45:16.649 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_Post❙post_engagement_toolbar_transition_key❙post_engagement_toolbar.eml|ddc981ccdd1c3fd9❙post_content.eml|e73333795cadcc57❙post_header.eml|2dce4b70ac02cd14❙id.ui.backstage.original_post❙
02-21 23:45:16.651 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.651 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.651 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|
02-21 23:45:16.651 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_menu.eml|cf95372aa71af81f❙>%G2❙Key_PostMetadataWithController❙post_metadata.eml|8fdafefdca415443❙&K}6❙post_avatar.eml|32d70da7bae26a07❙
02-21 23:45:16.652 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.652 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.652 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|
02-21 23:45:16.652 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_menu.eml|cf95372aa71af81f❙>%G2❙Key_PostMetadataWithController❙post_metadata.eml|8fdafefdca415443❙&K}6❙post_avatar.eml|32d70da7bae26a07❙
02-21 23:45:16.652 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.652 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.652 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|ContainerType|
02-21 23:45:16.652 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_menu.eml|cf95372aa71af81f❙>%G2❙Key_PostMetadataWithController❙post_metadata.eml|8fdafefdca415443❙&K}6❙post_avatar.eml|32d70da7bae26a07❙
02-21 23:45:16.653 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.653 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.653 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|ContainerType|
02-21 23:45:16.653 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_menu.eml|cf95372aa71af81f❙>%G2❙Key_PostMetadataWithController❙post_metadata.eml|8fdafefdca415443❙&K}6❙post_avatar.eml|32d70da7bae26a07❙
02-21 23:45:16.654 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.654 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.654 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|
02-21 23:45:16.654 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_menu.eml|cf95372aa71af81f❙>%G2❙Key_PostMetadataWithController❙post_metadata.eml|8fdafefdca415443❙&K}6❙post_avatar.eml|32d70da7bae26a07❙
02-21 23:45:16.654 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.654 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.654 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|
02-21 23:45:16.654 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_menu.eml|cf95372aa71af81f❙>%G2❙Key_PostMetadataWithController❙post_metadata.eml|8fdafefdca415443❙&K}6❙post_avatar.eml|32d70da7bae26a07❙
02-21 23:45:16.655 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.655 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.655 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|ContainerType|post_avatar.eml|32d70da7bae26a07|
02-21 23:45:16.655 16424 16424 D revanced: LithoFilterPatch: BufferStrings: avatar.eml|d00bf20757f77f85❙pG*)❙
02-21 23:45:16.656 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.656 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.656 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|ContainerType|post_avatar.eml|32d70da7bae26a07|avatar.eml|d00bf20757f77f85|ImageType|
02-21 23:45:16.656 16424 16424 D revanced: LithoFilterPatch: BufferStrings: eml.avatar❙Near❙UC8DIU56jF1ev5c6ubG_2IEQ"❙/@NearOTP❙//yt3.googleusercontent.com/dS8dawwcb27eQxwOfmMF_1afa79XqSdAVLkAHA3ZfGq3FQqjpZUgk5gsnKYSuj5815b_7cJ9=s68-c-k-c0x00ffffff-no-rj-mo❙
02-21 23:45:16.658 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.658 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.658 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|
02-21 23:45:16.658 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_PostMetadataWithController❙Key_SubscribeButtonWithController❙subscribe_button.eml|2b586d9a1ec5ecbc❙view_metadata.eml|79c841cbc5275ad1❙post_author.eml|d5eb007d5a08e27b❙
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_PostMetadataWithController❙Key_SubscribeButtonWithController❙subscribe_button.eml|2b586d9a1ec5ecbc❙view_metadata.eml|79c841cbc5275ad1❙post_author.eml|d5eb007d5a08e27b❙
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_PostMetadataWithController❙Key_SubscribeButtonWithController❙subscribe_button.eml|2b586d9a1ec5ecbc❙view_metadata.eml|79c841cbc5275ad1❙post_author.eml|d5eb007d5a08e27b❙
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_PostMetadataWithController❙Key_SubscribeButtonWithController❙subscribe_button.eml|2b586d9a1ec5ecbc❙view_metadata.eml|79c841cbc5275ad1❙post_author.eml|d5eb007d5a08e27b❙
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|ContainerType|
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_PostMetadataWithController❙Key_SubscribeButtonWithController❙subscribe_button.eml|2b586d9a1ec5ecbc❙view_metadata.eml|79c841cbc5275ad1❙post_author.eml|d5eb007d5a08e27b❙
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|ContainerType|
02-21 23:45:16.659 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_PostMetadataWithController❙Key_SubscribeButtonWithController❙subscribe_button.eml|2b586d9a1ec5ecbc❙view_metadata.eml|79c841cbc5275ad1❙post_author.eml|d5eb007d5a08e27b❙
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_PostMetadataWithController❙Key_SubscribeButtonWithController❙subscribe_button.eml|2b586d9a1ec5ecbc❙view_metadata.eml|79c841cbc5275ad1❙post_author.eml|d5eb007d5a08e27b❙
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|ContainerType|post_author.eml|d5eb007d5a08e27b|ContainerType|
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: BufferStrings: UC8DIU56jF1ev5c6ubG_2IEQ"❙/@NearOTP❙Near❙sans-serif-medium❙
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|ContainerType|post_author.eml|d5eb007d5a08e27b|ContainerType|TextType|
02-21 23:45:16.660 16424 16424 D revanced: LithoFilterPatch: BufferStrings: UC8DIU56jF1ev5c6ubG_2IEQ"❙/@NearOTP❙Near❙sans-serif-medium❙
02-21 23:45:16.661 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.661 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.661 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|view_metadata.eml|79c841cbc5275ad1|ContainerType|
02-21 23:45:16.661 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_time_text.eml|f486051b3073aaa7❙
02-21 23:45:16.661 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.661 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.661 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|view_metadata.eml|79c841cbc5275ad1|ContainerType|
02-21 23:45:16.661 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_time_text.eml|f486051b3073aaa7❙
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|view_metadata.eml|79c841cbc5275ad1|ContainerType|post_time_text.eml|f486051b3073aaa7|ContainerType|
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: BufferStrings: id.ui.backstage.published_time❙1 hour ago❙sans-serif-regular❙
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|view_metadata.eml|79c841cbc5275ad1|ContainerType|post_time_text.eml|f486051b3073aaa7|ContainerType|TextType|
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: BufferStrings: id.ui.backstage.published_time❙1 hour ago❙sans-serif-regular❙
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|subscribe_button.eml|2b586d9a1ec5ecbc|
02-21 23:45:16.662 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_SubscribeButtonWithController❙compact_subscribe_button.eml|c4192876c672d4b7❙
02-21 23:45:16.663 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.663 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.663 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|subscribe_button.eml|2b586d9a1ec5ecbc|compact_subscribe_button.eml|c4192876c672d4b7|ContainerType|
02-21 23:45:16.663 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_SubscribeButtonWithController❙Subscribe❙sans-serif-medium❙post_lockup❙post_lockup"❙eml.compact_subscribe_button❙Subscribe to Near.❙(EhhVQzhESVU1NmpGMWV2NWM2dWJHXzJJRVEgMygB❙(EhhVQzhESVU1NmpGMWV2NWM2dWJHXzJJRVEgMygB❙post_lockup❙UC8DIU56jF1ev5c6ubG_2IEQ❙EgIICxgA"^❙(EhhVQzhESVU1NmpGMWV2NWM2dWJHXzJJRVEgMygB❙(EhhVQzhESVU1NmpGMWV2NWM2dWJHXzJJRVEgMygB❙
02-21 23:45:16.663 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.663 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.663 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType|ContainerType|subscribe_button.eml|2b586d9a1ec5ecbc|compact_subscribe_button.eml|c4192876c672d4b7|ContainerType|TextType|
02-21 23:45:16.663 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_SubscribeButtonWithController❙Subscribe❙sans-serif-medium❙post_lockup❙post_lockup"❙eml.compact_subscribe_button❙Subscribe to Near.❙(EhhVQzhESVU1NmpGMWV2NWM2dWJHXzJJRVEgMygB❙(EhhVQzhESVU1NmpGMWV2NWM2dWJHXzJJRVEgMygB❙post_lockup❙UC8DIU56jF1ev5c6ubG_2IEQ❙EgIICxgA"^❙(EhhVQzhESVU1NmpGMWV2NWM2dWJHXzJJRVEgMygB❙(EhhVQzhESVU1NmpGMWV2NWM2dWJHXzJJRVEgMygB❙
02-21 23:45:16.664 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.664 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.664 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_menu.eml|cf95372aa71af81f|ImageType|
02-21 23:45:16.664 16424 16424 D revanced: LithoFilterPatch: BufferStrings: id.ui.backstage.post_menu_button❙Action menu❙Report❙Report❙GiRVZ2t4ZjJvQVBjUjB4OHlqR1F1QmlMbmtjRUVZcExUOTRTUmIoAjpECAEQBBokVWdreGYyb0FQY1IweDh5akdRdUJpTG5rY0VFWXBMVDk0U1JiWhhVQzhESVU1NmpGMWV2NWM2dWJHXzJJRVFKVggCEkQIARAEGiRVZ2t4ZjJvQVBjUjB4OHlqR1F1QmlMbmtjRUVZcExUOTRTUmJaGFVDOERJVTU2akYxZXY1YzZ1YkdfMklFURoAMgoIAyoGCJDY2K4GeAI%3D*❙Not interested❙Not interested❙AB9zfpLNavbYyRkwNKZ8AXDPNjDwiXWaylDoPYb9MsWPxNBb86hXWi2_JfUEjwdlQI9P8wGs2KypkeQmJ35S2ZRdYWBkIteu8x7IT3RORXZrw2gABuSyyAvRZ0KjHyPQvqjYop-sDjn1qs8UspxKy-BbUasX64bnt3YtnLOE5gUI4KIZ4x-rYFs❙Post removed❙Post removed❙Undo❙AB9zfpLAzjGpNw3mkOa6w7zXsIq0q_LLJsNicPjVbpQOWNUn6AY68Io1riXjmi8lCE952M67qpCge919zzab0st-tIMc3i9021QcP5fNlN5VBNlZ1r8T7tGK-JPdNFEOVx1KrU3QY8USxNS1TIxGcffDy0Y_Fc5xjZVmzgGbnmkqC6dtxu4StBI*❙"Don't recommend posts from channel❙"Don't recommend posts from channel❙AB9zfpLVEru4pwL8B4RFEEert0767AO3nBD_je0ta1Mmb6esrbjLO1CGoOKlXMjEUOiNfa6AEmE5mrAKmypM8xAugBdA2qKSWW1TcN3lOrgIr38dTKgYN_qqxsaqHO4xBkuhLGGMxiXD90nCX5jY_cmmt4Z8y-s__xDMQEIGtaN2xkLLi2lAhVBPChP0Hfajkf6CZyrtHw39❙7We won't recommend posts from this channel to you again❙7We won't recommend posts from this channel to you again❙Undo:❙AB9zfpI2WoRFFI4u5VBN6UiCodE2nodCX4acLLAAgRmbLYYRBUcbUPC_3YUwKZ7MBtOZHvE4Vtv6C2WgXBk7BG-sEnvagT4NBnXj0GcHCI5h07utomXbcfQzq5wvA24Pgb2m4HSBUtcHGmQ4P0UKE3VAGQBIBw13NwJ6mvNDzgDtRysITACVQHsxrNxmgVS2eiDoNO0XeoE2j❙Learn morej❙1//support.google.com/youtube/answer/6342839?hl=en❙yt_outline_overflow_vertical_black_36❙
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_attachment_container.eml|11bbffd694603ff7❙
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_attachment_container.eml|11bbffd694603ff7❙
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|post_attachment_container.eml|11bbffd694603ff7|ContainerType|
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: BufferStrings: image_attachment.eml|1536dfd67705f813❙
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|post_attachment_container.eml|11bbffd694603ff7|ContainerType|ContainerType|
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: BufferStrings: image_attachment.eml|1536dfd67705f813❙
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|post_attachment_container.eml|11bbffd694603ff7|ContainerType|ContainerType|
02-21 23:45:16.665 16424 16424 D revanced: LithoFilterPatch: BufferStrings: image_attachment.eml|1536dfd67705f813❙
02-21 23:45:16.666 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.666 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.666 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|post_attachment_container.eml|11bbffd694603ff7|ContainerType|ContainerType|image_attachment.eml|1536dfd67705f813|ContainerType|
02-21 23:45:16.666 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Image attachment❙EmJodHRwczovL3l0My5nZ3BodC5jb20vMFoxcmlMTVJJcHRWWTY4RXFMWXlHaVBEUnhzVDZlQkFPZENaLWNZWG9UNEREQlpxaG5DN04ycEhsUUFaWXdnc3J3TGhPbzhfa2VqSyC5ASgB❙"2B0❙spinner_48p_grey❙mhttps://yt3.ggpht.com/0Z1riLMRIptVY68EqLYyGiPDRxsT6eBAOdCZ-cYXoT4DDBZqhnC7N2pHlQAZYwgsrwLhOo8_kejK=s288-nd-v1 ❙mhttps://yt3.ggpht.com/0Z1riLMRIptVY68EqLYyGiPDRxsT6eBAOdCZ-cYXoT4DDBZqhnC7N2pHlQAZYwgsrwLhOo8_kejK=s400-nd-v1 ❙mhttps://yt3.ggpht.com/0Z1riLMRIptVY68EqLYyGiPDRxsT6eBAOdCZ-cYXoT4DDBZqhnC7N2pHlQAZYwgsrwLhOo8_kejK=s512-nd-v1 ❙mhttps://yt3.ggpht.com/0Z1riLMRIptVY68EqLYyGiPDRxsT6eBAOdCZ-cYXoT4DDBZqhnC7N2pHlQAZYwgsrwLhOo8_kejK=s640-nd-v1 ❙mhttps://yt3.ggpht.com/0Z1riLMRIptVY68EqLYyGiPDRxsT6eBAOdCZ-cYXoT4DDBZqhnC7N2pHlQAZYwgsrwLhOo8_kejK=s680-nd-v1 ❙666666❙?!999999❙post-image-hint*❙L?*7"5Tap to zoom out, or pinch with two fingers to zoom in ❙
02-21 23:45:16.666 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.666 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.666 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|post_attachment_container.eml|11bbffd694603ff7|ContainerType|ContainerType|image_attachment.eml|1536dfd67705f813|ContainerType|277808098|
02-21 23:45:16.666 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Image attachment❙EmJodHRwczovL3l0My5nZ3BodC5jb20vMFoxcmlMTVJJcHRWWTY4RXFMWXlHaVBEUnhzVDZlQkFPZENaLWNZWG9UNEREQlpxaG5DN04ycEhsUUFaWXdnc3J3TGhPbzhfa2VqSyC5ASgB❙"2B0❙spinner_48p_grey❙mhttps://yt3.ggpht.com/0Z1riLMRIptVY68EqLYyGiPDRxsT6eBAOdCZ-cYXoT4DDBZqhnC7N2pHlQAZYwgsrwLhOo8_kejK=s288-nd-v1 ❙mhttps://yt3.ggpht.com/0Z1riLMRIptVY68EqLYyGiPDRxsT6eBAOdCZ-cYXoT4DDBZqhnC7N2pHlQAZYwgsrwLhOo8_kejK=s400-nd-v1 ❙mhttps://yt3.ggpht.com/0Z1riLMRIptVY68EqLYyGiPDRxsT6eBAOdCZ-cYXoT4DDBZqhnC7N2pHlQAZYwgsrwLhOo8_kejK=s512-nd-v1 ❙mhttps://yt3.ggpht.com/0Z1riLMRIptVY68EqLYyGiPDRxsT6eBAOdCZ-cYXoT4DDBZqhnC7N2pHlQAZYwgsrwLhOo8_kejK=s640-nd-v1 ❙mhttps://yt3.ggpht.com/0Z1riLMRIptVY68EqLYyGiPDRxsT6eBAOdCZ-cYXoT4DDBZqhnC7N2pHlQAZYwgsrwLhOo8_kejK=s680-nd-v1 ❙666666❙?!999999❙post-image-hint*❙L?*7"5Tap to zoom out, or pinch with two fingers to zoom in ❙
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|post_engagement_toolbar.eml|ddc981ccdd1c3fd9|ContainerType|
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_comment_button.eml|285fe3c99a0f9574❙PDSA❙Key_PostLikeDislike_2❙post_like_dislike.eml|180ea11426f50df0❙
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|post_engagement_toolbar.eml|ddc981ccdd1c3fd9|ContainerType|ContainerType|
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_comment_button.eml|285fe3c99a0f9574❙PDSA❙Key_PostLikeDislike_2❙post_like_dislike.eml|180ea11426f50df0❙
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|post_engagement_toolbar.eml|ddc981ccdd1c3fd9|ContainerType|ContainerType|
02-21 23:45:16.667 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_comment_button.eml|285fe3c99a0f9574❙PDSA❙Key_PostLikeDislike_2❙post_like_dislike.eml|180ea11426f50df0❙
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|post_engagement_toolbar.eml|ddc981ccdd1c3fd9|ContainerType|
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_comment_button.eml|285fe3c99a0f9574❙PDSA❙Key_PostLikeDislike_2❙post_like_dislike.eml|180ea11426f50df0❙
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|post_engagement_toolbar.eml|ddc981ccdd1c3fd9|ContainerType|ContainerType|
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: BufferStrings: post_comment_button.eml|285fe3c99a0f9574❙PDSA❙Key_PostLikeDislike_2❙post_like_dislike.eml|180ea11426f50df0❙
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|post_engagement_toolbar.eml|ddc981ccdd1c3fd9|ContainerType|post_like_dislike.eml|180ea11426f50df0|ContainerType|
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_PostLikeDislike_2❙post_dislike_button.eml|c551032394fc4336❙post_like_count.eml|beec6240a5b6a084❙Key_PostAnimatedLikeButton❙post_animated_like_button.eml|9e889813611f0676❙Q~I`❙Le)Z❙
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|post_engagement_toolbar.eml|ddc981ccdd1c3fd9|ContainerType|post_like_dislike.eml|180ea11426f50df0|ContainerType|
02-21 23:45:16.668 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_PostLikeDislike_2❙post_dislike_button.eml|c551032394fc4336❙post_like_count.eml|beec6240a5b6a084❙Key_PostAnimatedLikeButton❙post_animated_like_button.eml|9e889813611f0676❙Q~I`❙Le)Z❙
02-21 23:45:16.669 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.669 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.669 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|post_engagement_toolbar.eml|ddc981ccdd1c3fd9|ContainerType|post_like_dislike.eml|180ea11426f50df0|ContainerType|
02-21 23:45:16.669 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_PostLikeDislike_2❙post_dislike_button.eml|c551032394fc4336❙post_like_count.eml|beec6240a5b6a084❙Key_PostAnimatedLikeButton❙post_animated_like_button.eml|9e889813611f0676❙Q~I`❙Le)Z❙
02-21 23:45:16.669 16424 16424 D revanced: LithoFilterPatch: Searching 
02-21 23:45:16.669 16424 16424 D revanced: LithoFilterPatch: ID: null
02-21 23:45:16.669 16424 16424 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|ContainerType|post_engagement_toolbar.eml|ddc981ccdd1c3fd9|ContainerType|post_like_dislike.eml|180ea11426f50df0|ContainerType|
02-21 23:45:16.669 16424 16424 D revanced: LithoFilterPatch: BufferStrings: Key_PostLikeDislike_2❙post_dislike_button.eml|c551032394fc4336❙post_like_count.eml|beec6240a5b6a084❙Key_PostAnimatedLikeButton❙post_animated_like_button.eml|9e889813611f0676❙Q~I`❙Le)Z❙
```

**Comment by oSumAtrIX** [comment]
The one you described

**Comment by ILoveOpenSourceApplications** [comment]
Does this mean something?
```kotlin
02-22 00:06:37.729 24919 25430 D revanced: LithoFilterPatch: BufferStrings: id.ui.backstage.post_menu_button❙Action menu❙Report❙Report❙GiRVZ2t4YS1BNUd6b3JISDduZ0g5eFBUbkRGNnpWeGJZRHhTMlcoAjpECAEQBBokVWdreGEtQTVHem9ySEg3bmdIOXhQVG5ERjZ6VnhiWUR4UzJXWhhVQ0JpMm1yV3VOdXlZeTRnYk02ZlUxOFFK-AQIAhJECAEQBBokVWdreGEtQTVHem9ySEg3bmdIOXhQVG5ERjZ6VnhiWUR4UzJXWhhVQ0JpMm1yV3VOdXlZeTRnYk02ZlUxOFEaoQQSngRNb3JlIHRoYW4gdHdvIHllYXJzIGFmdGVyIGNpbmVtYXRvZ3JhcGhlciBIYWx5bmEgSHV0Y2hpbnMgd2FzIGZhdGFsbHkgc2hvdCBvbiB0aGUgTmV3IE1leGljbyBzZXQgb2YgIlJ1c3QsIiB0aGUgZmlsbSdzIHdlYXBvbnMgaGFuZGxlciBpcyBzZXQgdG8gZ28gb24gdHJpYWwgaW4gdGhlIGhpZ2gtcHJvZmlsZSBjYXNlLgoKQXJtb3JlciBIYW5uYWggR3V0aWVycmV6IHdhcyBjaGFyZ2VkIHdpdGggdHdvIGNvdW50cyBvZiBpbnZvbHVudGFyeSBtYW5zbGF1Z2h0ZXIgaW4gY29ubmVjdGlvbiB3aXRoIHRoZSBPY3RvYmVyIDIwMjEgb24tc2V0IGRlYXRoLiAKClNoZSBmYWNlcyBhbiBhZGRpdGlvbmFsIGNoYXJnZSBvZiB0YW1wZXJpbmcgd2l0aCBldmlkZW5jZSwgd2l0aCBzdGF0ZSBwcm9zZWN1dG9ycyBjbGFpbWluZyBzaGUgaGFuZGVkIG9mZiBhIHNtYWxsIGJhZyBvZiBjb2NhaW5lIGZvbGxvd2luZyBoZXIgaW50ZXJ2aWV3IHdpdGggcG9saWNlIHRoZSBkYXkgb2YgdGhlIHNob290aW5nLiBodHRwczovL2FiY25ld3MudmlzaXRsaW5rLm1lL1c2X2VzcjIKCAMqBgiP49iuBngC*❙Not interested❙Not interested❙AB9zfpIu-2i6aQF0V8aQ0QLHUI6fXZ65QjZ8GzEZGw9G-42y8yanF4NfU04aIpFu35e6S3VGsSW4W8sbaPelThhhrfxA2n8z9a5U730AJXctHlClR18XiUfzzYw533WFuL1gCQ8RvgFkjxM62UuGEH44tG4OAWMC1cr99PgO7W6ZjTePLiDJLQU❙Post removed❙Post removed❙Undo❙AB9zfpL4b0V1VFKjLZHfgP9Ve8Kf-bGysfrInJ8RgoX96WDi-_bXn-B74LvZ_qgoDhAD8GeTwb9Bo9-fPdVBaiaEMuqI-ICFabciGYOVpHsgwhPVoejvmuQqYoeL3yQ8N69wzXbret3DMtMXzDtjHRFGTAef1F1MBnVpBBHOm2Uarji0N8JqtPo*❙"Don't recommend posts from channel❙"Don't recommend posts from channel❙AB9zfpJdSNTppW0txmzVOClkAb5PTlkz3qwlf22lePxMuruTqwFp40Gxhfra27XyG74vhwBBpYMT9yJoYLcZwzNxiIE_GNrKnjR-nPHBe7SoJHkpOxDN7a1HaHyf05LEK6BbfQfV1N3Ycl_n0Ur1FlDPaL5Q87P2kp4vzcM4ciklHhineCRrWVvQur0BONDSSbnLG7GrEgKq❙7We won't recommend posts from this channel to you again❙7We won't recommend posts from this channel to you again❙Undo:❙AB9zfpKOnOAqoDeyuStOOfO01_6SMx1bGgHS_t7JOhyeh5TVkxU1Yo7m9Jx2whkOG_agw2kPsAQqOp_8Rg5Aa9KPpztW9yLV82__QC2FSzrt9NXqb6vajg-AKrBc5gvMnPuITxsAS3gC_8eZvJF6Qu7-2dE5lg3XytSI7JjDmhTeY4-XyZ7tu0n19glSzhyeOElQLhITQ8mBj❙Learn morej❙1//support.google.com/youtube/answer/6342839?hl=en❙yt_outline_overflow_vertical_black_36❙

```

**Comment by oSumAtrIX** [comment]
I don't think so

**Comment by oSumAtrIX** [comment]
You can try blocking it and confirm yourself

**Comment by ILoveOpenSourceApplications** [comment]
I think that's the 3 dot menu on the side, hence the report, not interested and so forth options. Also, that's not a .eml file, so will it be blocked by custom filter?

**Comment by oSumAtrIX** [comment]
I suggest to try and see

**Comment by ILoveOpenSourceApplications** [comment]
```kotlin
02-22 00:20:39.011 10626 11339 D revanced: Filter: LayoutComponentsFilter Filtered identifier: cell_divider.eml|a3c76cf45290d34f
02-22 00:20:39.662 10626 11339 D revanced: LithoFilterPatch: Searching 
02-22 00:20:39.662 10626 11339 D revanced: LithoFilterPatch: ID: post_base_wrapper.eml|5e2b195a9a5d7880
02-22 00:20:39.662 10626 11339 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|
02-22 00:20:39.662 10626 11339 D revanced: LithoFilterPatch: BufferStrings: expand-post-cell❙post_base.eml|397e38aba64a8981❙&g_{❙@EiRVZ2t4QW1zMHdjc2tEZW53dDBUZ1FfeHBFN3FKdGFaM084VUwgqAIoAQ%3D%3D❙".B,❙1708541439501371592❙post_base_wrapper.eml|5e2b195a9a5d7880❙
02-22 00:20:39.665 10626 11338 D revanced: LithoFilterPatch: BufferStrings: Key_Post❙post.eml|b9dd6295ab418a5f❙id.ui.backstage.post❙
02-22 00:20:39.666 10626 11339 D revanced: LithoFilterPatch: Searching 
02-22 00:20:39.666 10626 11339 D revanced: LithoFilterPatch: ID: null
02-22 00:20:39.666 10626 11339 D revanced: LithoFilterPatch: Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|
02-22 00:20:39.667 10626 11338 D revanced: LithoFilterPatch: BufferStrings: Key_Post❙post_engagement_toolbar_transition_key❙post_engagement_toolbar.eml|ddc981ccdd1c3fd9❙post_content.eml|e73333795cadcc57❙post_header.eml|2dce4b70ac02cd14❙id.ui.backstage.original_post❙

```

**Comment by ILoveOpenSourceApplications** [comment]
> Maybe try:
> 
> `^post_base_wrapper.eml$id.ui.backstage.post`

For some reason after I tried to filter this one, I'm not getting any community posts to show up in mu home feed.

**Comment by ILoveOpenSourceApplications** [comment]
> Is the filter working? It will show in the logs when custom filter hid stuff.

For me to know that wouldn't I need to know what element needs to be hidden in the first place, given that we're doing a trial n error method

**Comment by ILoveOpenSourceApplications** [comment]
But like I said, I'm not seeing any posts in my home feed, and if there's no posts, how would the filter capture it as hidden?

**Comment by ILoveOpenSourceApplications** [comment]
```kotlin
02-22 01:43:17.803 20862 21059 D revanced: LithoFilterPatch: Proto buffer is null, using an empty buffer array
02-22 01:43:17.803 20862 21059 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType| BufferStrings: 
02-22 01:43:17.803 20862 21059 D revanced: LithoFilterPatch: Proto buffer is null, using an empty buffer array
02-22 01:43:17.803 20862 21059 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType| BufferStrings: 
02-22 01:43:17.803 20862 21060 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14
```

**Comment by ILoveOpenSourceApplications** [comment]
Can this one be a component expand-post-cell❙post_base.eml?

**Comment by ILoveOpenSourceApplications** [comment]
Does this log help in anyways?

```kotlin
02-22 02:17:40.728 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|post_attachment_container.eml|11bbffd694603ff7|ContainerType| BufferStrings: image_attachment.eml|1536dfd67705f813❙
02-22 02:17:40.728 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|post_attachment_container.eml|11bbffd694603ff7|ContainerType|ContainerType| BufferStrings: image_attachment.eml|1536dfd67705f813❙
02-22 02:17:40.728 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|post_attachment_container.eml|11bbffd694603ff7|ContainerType|ContainerType| BufferStrings: image_attachment.eml|1536dfd67705f813❙
02-22 02:17:40.728 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_menu.eml|cf95372aa71af81f|ImageType| BufferStrings: id.ui.backstage.post_menu_button❙Action menu❙Report❙Report❙GiRVZ2t4MF9pMllaZ2JNN2V3dG9WQXgtOF8wRm5zOEk0LU5UOTAoAjpECAEQBBokVWdreDBfaTJZWmdiTTdld3RvVkF4LThfMEZuczhJNC1OVDkwWhhVQ0JpMm1yV3VOdXlZeTRnYk02ZlUxOFFKhQMIAhJECAEQBBokVWdreDBfaTJZWmdiTTdld3RvVkF4LThfMEZuczhJNC1OVDkwWhhVQ0JpMm1yV3VOdXlZeTRnYk02ZlUxOFEargISqwJKVVNUIElOOiBBIHlvdW5nIGdpcmwgaGFzIGRpZWQgYW5kIGEgYm95IGhhcyBiZWVuIGhvc3BpdGFsaXplZCBhZnRlciB0aGV5IGZlbGwgaW50byBhIGhvbGUgaW4gdGhlIHNhbmQgYXQgYSBiZWFjaCBUdWVzZGF5IGF0IExhdWRlcmRhbGUtYnktdGhlLVNlYSwgRmxvcmlkYS4KClRoZSB0d28gY2hpbGRyZW4gd2VyZSBwdWxsZWQgb3V0IG9mIHRoZSBzYW5kIGFuZCB0YWtlbiB0byB0aGUgaG9zcGl0YWwsIGJ1dCBwb2xpY2UgbGF0ZXIgY29uZmlybWVkIHRoZSBnaXJsIGhhZCBkaWVkLiBodHRwczovL3RyaWIuYWwvYUpxVmtuZTIKCAMqBgjB7NiuBngC*❙Not interested❙Not interested❙AB9zfpJoTmeeUPR7Cn9QJae3AP7ltgJwmSA5yAfVrYkw9xtb2Vt9IBMgn-QtvALxrnNFjTIzLfmOdCTRoIm5BUk5ISpA9oKOyFelAYU-iHPfFHRDAurQRZXuoXUbF6Po5l5H_AWfrvO-qQXdIZFzpsLXS20hRWQZtU16jBpqcQmORGn56qPnm7E❙Post removed❙Post removed❙Undo❙AB9zfpI4fsS-RgaxMMQlPRGmXNlFfjqLkisG_-Sak75eWYx5C7yG1EIx7msMCi7hVQ2CBMFia6HLGe126yLBHi9bbC3Y78JOvShVbeXD6owNAiglgCgOvGuV9S9-bT9UgY9g0Ohx8Sz4R9uHJmwoMB6jDiAnfEVVfFXD7xCSTolDUiQ8yjNEcAY*❙"Don't recommend posts from channel❙"Don't recommend posts from channel❙AB9zfpJKVLcglDSyQj2RmHvaazwc5RK_BhhPQKwNLMeeF-PvF0eNtfs8R6DVQO30N1kSloOQFPqx5x8ExG1OmvXlM1EEVym60hkn0AJq1zB0h674wTddVyGgjx0O0FKlGyl16GEz38IukOUsDdDt7fcW9aTdu5nat7uzFjkH9zU2_miB1J_nNWYp-DI1wIe7vBbPPG4yTb5d❙7We won't recommend posts from this channel to you again❙7We won't recommend posts from this channel to you again❙Undo:❙AB9zfpLXHgJ7KEmHLPM5weNIy_2kUUEthlnhOOuhaW4wLccetU66G5GnLHwGf4ElhU3J3GcD8ugeSkRyIlPxU8V9SzSdATyb5rgNyIMZIsfdxVj1S2_3TQtCV0HpTQpmIG1z8uJxBjUqsR7ZOgunV1BaasLHfkHFcNjdgP_LD7hp_P4q1k1Wo6th9KcslKmi8JQW1ZqYNsCTj❙Learn morej❙1//support.google.com/youtube/answer/6342839?hl=en❙yt_outline_overflow_vertical_black_36❙
02-22 02:17:40.729 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_header.eml|2dce4b70ac02cd14|ContainerType|ContainerType|post_metadata.eml|8fdafefdca415443|ContainerType| BufferStrings: Key_PostMetadataWithController❙Key_SubscribeButtonWithController❙subscribe_button.eml|2b586d9a1ec5ecbc❙view_metadata.eml|79c841cbc5275ad1❙d\xcQz❙post_author.eml|d5eb007d5a08e27b❙#$ =❙

```

**Comment by ILoveOpenSourceApplications** [comment]
These lines are having "view all posts" in them.
```kotlin
02-22 02:17:40.726 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|button.eml|4bb30323fafcaaee|ContainerType| BufferStrings: View all posts❙sans-serif❙View all posts❙FEpost_home❙
02-22 02:17:40.726 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|button.eml|4bb30323fafcaaee|ContainerType|TextType| BufferStrings: View all posts❙sans-serif❙View all posts❙FEpost_home❙
```

**Comment by ILoveOpenSourceApplications** [comment]
```kotlin
02-22 02:17:40.725 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|ContainerType| BufferStrings: post_attachment_container.eml|11bbffd694603ff7❙post_text.eml|1fcd84491683c5ea❙
02-22 02:17:40.725 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|ContainerType|ContainerType| BufferStrings: post_attachment_container.eml|11bbffd694603ff7❙post_text.eml|1fcd84491683c5ea❙
02-22 02:17:40.725 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType|ContainerType|ContainerType| BufferStrings: post_attachment_container.eml|11bbffd694603ff7❙post_text.eml|1fcd84491683c5ea❙
02-22 02:17:40.725 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|post_content.eml|e73333795cadcc57|ContainerType| BufferStrings: post_attachment_container.eml|11bbffd694603ff7❙post_text.eml|1fcd84491683c5ea❙
02-22 02:17:40.726 12826 13380 D revanced: LithoFilterPatch: Searching ID: null Path: home_video_with_context.eml|94d72f4dbe072479|CellType|video_with_context_slots.eml|735d981756332ff7|ContainerType|ContainerType|metadata.eml|774aa1e2701ce9e7| BufferStrings: metadata_content.eml|36003a8802392cbd❙
02-22 02:17:40.726 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|button.eml|4bb30323fafcaaee|ContainerType| BufferStrings: View all posts❙sans-serif❙View all posts❙FEpost_home❙
02-22 02:17:40.726 12826 12982 D revanced: LithoFilterPatch: Searching ID: null Path: post_base_wrapper.eml|5e2b195a9a5d7880|CellType|post_base.eml|397e38aba64a8981|ContainerType|post.eml|b9dd6295ab418a5f|ContainerType|ContainerType|button.eml|4bb30323fafcaaee|ContainerType|TextType| BufferStrings: View all posts❙sans-serif❙View all posts❙FEpost_home❙
02-22 02:17:40.726 12826 13380 D revanced: LithoFilterPatch: Searching ID: null Path: home_video_with_context.eml|94d72f4dbe072479|CellType|video_with_context_slots.eml|735d981756332ff7|ContainerType|ContainerType|metadata.eml|774aa1e2701ce9e7|metadata_content.eml|36003a8802392cbd|ContainerType| BufferStrings: UndeniableLuck ❙ 3.5M views ❙ 1 year ago❙sans-serif-regular❙Funniest KSI Try Not to Laugh Moments❙sans-serif-medium❙eml.metadata❙
```

**Comment by ILoveOpenSourceApplications** [comment]
> It's still hard to see what's going on there.
> 
> Can try: `^post_base_wrapper.eml$post_attachment_container.eml`

This one hides the image in the post.

> Or `^post_base_wrapper.eml$FEpost_home`


> Or the English specific filter of: `^post_base_wrapper.eml$View all posts`

Both of these worked, as the button is gone. But there's still the animation for the button popping in.

**Comment by ILoveOpenSourceApplications** [comment]
See how when the button pops in by pushing down the video below it. Even if the button is hidden, the post pushes down the video below it to show more black space. 

https://github.com/ReVanced/revanced-patches/assets/117499019/718d4ddb-298c-4ab4-97a3-c358bfcf1e94

**Comment by ILoveOpenSourceApplications** [comment]
A solution for the animation would be to bring the preview comment box which also appears similar to the view all posts button. But I'm unsure if this is even possible or not, but just putting it out there when I thought about it.

**Comment by oSumAtrIX** [comment]
Over time, a lot of feature requests have been made, which is why I am closing some according to the [contribution guidelines](https://github.com/ReVanced/revanced-patches/blob/main/CONTRIBUTING.md#-submitting-a-feature-request):

> Requests can be accepted or rejected at the discretion of maintainers of ReVanced Patches

This feature request has been closed for one or more of the following reasons:

- Stale for too long
- Low priority
- Maintainers un-interested
- Too complex to realize

Regardless, anyone who comes across this feature request can PR it. Thanks for the feature request!
